scaleway/helpers_lb.go

```diff
@@ -207,3 +207,29 @@ func expandLbHCHTTPS(raw interface{}) *lb.HealthCheckHTTPSConfig {
 		Code:   expandInt32Ptr(rawMap["code"]),
 	}
 }
+
+func expandLbCertificateLetsEncrypt(raw interface{}) *lb.CreateCertificateRequestLetsencryptConfig {
```

Comment:
What do you think about `expandLbLetsEncrypt`?

Reply:
Fine by me